Students/Neacsu Ioana - Alina/Lab4/src/POOlab4/Order.java

```diff
@@ -0,0 +1,81 @@
+package POOlab4;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class Order {
+    private String ID;
+    private Customer customer;
+    private List<OrderItem> orderList;
+    private int orderAmount;
+    public Order() {
+        this.orderList = new ArrayList<>();
+    }
+
+    public Order(Customer customer) {
+        this.customer = customer;
+        this.orderList = new ArrayList<>();
+    }
+
+    public Customer getCustomer() {
+        return customer;
+    }
+
+    public void setCustomer(Customer customer) {
+        this.customer = customer;
+    }
+
+    public String getID() {
+        return ID;
+    }
+
+    public void setID(String ID) {
+        this.ID = ID;
+    }
+
+    public List<OrderItem> getOrderList() {
+        return orderList;
+    }
+
+    public void setOrderList(List<OrderItem> orderList) {
+        this.orderList = orderList;
+    }
+
+    public void updateAmount(int amount){
+        orderAmount += amount;
+    }
+    public void addProductToOrder(Order order, Product product, int quantity) {
```

Comment:
primul parametru e cam in plus aici. Tu ca sa poti folosi metoda asta ai nevoie de o instanta din clasa asta si atunci vei face ceva de genul dupa ce initializezi obiectul `order.addProductToOrder(order, product, etc)`. In aceasta situatie in metoda de aici in loc de primul parametru puteai folosi `this` in interiorul metodei.